make/compiler/Makefile.pgi

```diff
@@ -51,6 +51,8 @@ GEN_DYNAMIC_FLAG = -Bdynamic
 LIB_STATIC_FLAG =
 LIB_DYNAMIC_FLAG = -shared
 
+# TODO: set target architecture for optimization
+
```

Comment:
Not your fault, but it looks like we only have the IEEE_FLOAT_GEN_CFLAGS for pgi. We should also have `NO_IEEE_FLOAT_GEN_CFLAGS = -Knoieee`. pgi's default is `Knoieee` but we should have both for cases like `chpl test.chpl --no-ieee-floats --ieee-floats`